low-level/sensors/impl/platforms/realstor/realstor_logical_volume_sensor.py

```diff
@@ -163,11 +184,19 @@ def run(self):
         self._read_my_msgQ_noWait()
 
         disk_groups = None
+        logical_volumes = None
+
         try:
             disk_groups = self._get_disk_groups()
```

Comment:
do we need this now ? We can stop invoking this code, though func can be maintained as it is.

Reply:
trying to reuse code today. but later in LDR R2 we need segregation for Disk group & logical volumes totatlly, when disk group alerts are introduced. Fine lets that for later only.